game/views.py

```
@@ -579,6 +604,43 @@ def verify_otp(request):
     return render(request, 'game/verify_otp.html')
 
 
+
+class CustomPasswordResetView(PasswordResetView):
+
+    def post(self, request, *args, **kwargs):
```

Ruff (0.15.13)

[warning] Missing type annotation for `*args`

(ANN002)